internal/usage/manager.go

```diff
@@ -16,8 +17,8 @@ type FeatureUsage map[string]models.Usage
 
 //go:generate mockgen -source=manager.go -package=usage -destination=mock_usage_manager.go
 type API interface {
-	Add(usages FeatureUsage, name string, data *map[string]interface{})
```

Comment:
I think you are wrong. The name is still there, with same string. I called the variable featureKey because its not just the name inside each Usage, but also the key in the Usage map.

Reply:
Changed the variable name back to `name` because of confusion.